apps/image-editor/src/js/action.js

```diff
@@ -405,6 +406,83 @@ export default {
     );
   },
 
+  /**
+   * Resize Action
+   * @returns {Object} actions for ui resize
+   * @private
+   */
+  _resizeAction() {
+    return extend(
+      {
+        getCurrentDimensions: () => this._graphics.getCurrentDimensions(),
+        preview: (actor, value, lockState) => {
+          const currentDimensions = this._graphics.getCurrentDimensions();
+          const calcAspectRatio = () => currentDimensions.width / currentDimensions.height;
+
+          let dimensions = {};
+          switch(actor) {
+            case 'width':
+              dimensions.width = value;
+              if (lockState) {
+                dimensions.height = value / calcAspectRatio();
+              } else {
+                dimensions.height = currentDimensions.height;
+              }
+              break;
+            case 'height':
+              dimensions.height = value;
+              if (lockState) {
+                dimensions.width = value * calcAspectRatio();
+              } else {
+                dimensions.width = currentDimensions.width;
+              }
```

Comment:
When lockState is turned on, validation should be performed when the size is calculated.

This is because it exceeds 4088.

Reply:
Added validation

Reply:
I checked validation.
However, When operated in conjunction with the zoom operation, a side effect occurs.

<img width="568" alt="스크린샷 2021-04-15 21 56 46" src="https://user-images.githubusercontent.com/43128697/114872983-caee4400-9e35-11eb-837c-e295a7e9d2e0.png">

@lja1018 Please leave helpful feedback regarding the zoom action.

Reply:
@seytar
It can be resolved by invoking an API that resets zoom prior to resize image.